packages/bpk-component-slider/src/BpkSlider.tsx

```diff
@@ -54,7 +54,7 @@ const BpkSlider = (props: Props) => {
       withTracks
       snapDragDisabled={false}
       invert={invert}
-      // TODO: className to be removed
+      // Accepted prop passed to the next component
```

Comment:
Can you pop the `allowed` components here: https://github.com/Skyscanner/backpack/blob/55a9215b29e2c164045e33f99a02bcf6295c69d9/.eslintrc#L131 using `allowedFor`?